torch/_utils_internal.py

```diff
@@ -52,5 +53,42 @@ def get_source_lines_and_file(obj):
     return sourcelines, file_lineno, filename
 
 
+def check_module_version_greater_or_equal(module, req_version_tuple, error_if_malformed=True):
```

Comment:
@zou3519 , do you think this is the right place for me to put this function?

Reply:
Given the comment at the top of the file:
```
# this arbitrary-looking assortment of functionality is provided here
# to have a central place for overrideable behavior. The motivating
```
I'm a bit scared to put this function in this file. I don't know what the comment means, but I don't think we're going to override this function in the "FB build environment" that it quotes.

Since we're only using the function in one place right now I'd be fine with tossing it into torch/serialization.py we could punt the question of "what file to put this helper function into" to if we get future uses in other files

Reply:
Ok makes sense. I moved the function to serialization.py.